tests/repository.py

```diff
@@ -140,69 +130,56 @@ class TestBasics(unittest.TestCase):
     """Test case for basic functions of the repository classes."""
 
     def setUp(self):
-        inputRoot = 'tests/butlerAlias'
-        outputRootA = 'tests/repository/repoA'
-        outputRootB = 'tests/repository/repoB'
-
-        storageCfg = dp.PosixStorage.cfg(root=inputRoot)
-        accessCfg = dp.Access.cfg(storageCfg=storageCfg)
-        inputRepoCfg = dp.Repository.cfg(accessCfg=accessCfg,
-                                         mapper=ParentMapper(ParentMapper.cfg(root=inputRoot)))
-
-        storageCfg = dp.PosixStorage.cfg(root=outputRootB)
-        accessCfg = dp.Access.cfg(storageCfg=storageCfg)
-        repoBCfg = dp.Repository.cfg(accessCfg=accessCfg, parentCfgs=[inputRepoCfg],
-                                     mapper=ChildrenMapper(root=outputRootB))
-
-        storageCfg = dp.PosixStorage.cfg(root=outputRootA)
-        accessCfg = dp.Access.cfg(storageCfg=storageCfg)
-        repoACfg = dp.Repository.cfg(accessCfg=accessCfg, mapper=ChildrenMapper(root=outputRootA),
-                                     peerCfgs=[repoBCfg], parentCfgs=[inputRepoCfg])
+        self.tearDown()
 
-        butlerCfg = dp.Butler.cfg(repoCfg=repoACfg)
-        self.butler = dp.Butler(butlerCfg)
-
-        self.datasetType = 'raw'
+        inputRepoArgs = dp.RepositoryArgs(root='tests/butlerAlias',
+                                          mapper=ParentMapper)
+        # mode of output repos is write-only by default
+        outputRepoArgs = dp.RepositoryArgs(root='tests/repository/repoA',
+                                           mapper=ChildrenMapper,
+                                           mode='rw')
+        self.butler = dp.Butler(inputs=inputRepoArgs, outputs=outputRepoArgs)
 
     def tearDown(self):
         if os.path.exists('tests/repository'):
             shutil.rmtree('tests/repository')
-        del self.butler
+        if os.path.exists('tests/butlerAlias/repositoryCfg.yaml'):
+            os.remove('tests/butlerAlias/repositoryCfg.yaml')
 
     def testGet(self):
-        raw_image = self.butler.get(self.datasetType, {'visit':'2', 'filter':'g'})
+        raw_image = self.butler.get('raw', {'visit':'2', 'filter':'g'})
         # in this case the width is known to be 1026:
         self.assertEqual(raw_image[1].header["NAXIS1"], 1026) # raw_image is an lsst.afw.ExposureU
```

Comment:
Not this changeset, but this comment is definitely wrong.  It looks like it may be a PyFITS/astropy.io.fits object.